controllers/secret_controller.go

```diff
@@ -462,6 +462,13 @@ func createSubroutes(namespaceList []string, receiver receiverType) *alertmanage
 		{Receiver: receiverNull, Match: map[string]string{"alertname": "ClusterOperatorDown", "name": "insights"}},
 	}
 
+	if !config.IsFedramp() {
+        // Route ClusterOperatorDown for monitoring to null receiver https://issues.redhat.com/browse/OSD-19769
+        subroute = append(subroute,
+			&alertmanager.Route{Receiver: receiverNull, Match: map[string]string{"alertname": "ClusterOperatorDown", "name": "monitoring"}},
+		)
+    }
+
```

Comment:
Any specific reason we aren't doing this for fedramp? The other alerts we routed to nullreceiver always made it into the generic subroute

Reply:
Correct me if I'm wrong, but we don't have visibility of those clusters, I'm not even sure they are being taken into consideration in the on-call relief effort metrics. We want to be able to tweak the alert so being able to review incoming alerts is a pre-requisite.